fpdf/fpdf.py

```diff
@@ -2029,28 +2048,38 @@ def cell(
                 "ignored"
             )
             border = 1
+        newpos_x = X.RIGHT
+        newpos_y = Y.TOP
+        if ln == 1:
+            newpos_x = X.LMARGIN
+            newpos_y = Y.NEXT
+        elif ln == 2:
+            newpos_x = X.LEFT
+            newpos_y = Y.NEXT
         # Font styles preloading must be performed before any call to FPDF.get_string_width:
         txt = self.normalize_text(txt)
         styled_txt_frags = self._preload_font_styles(txt, markdown)
         return self._render_styled_cell_text(
+            TextLine(styled_txt_frags, 0.0, 0, False),
```

Comment:
The `text_width`, `number_of_spaces_between_words` & `justify` parameters could be passed by name here, to improve readability.

Reply:
will do.